lib/iris/manager.txt

```diff
@@ -0,0 +1,94 @@
+_lazy_data.py
```

Comment:
Presumably this text file will be removed at some point. Or at least turned into some documentation??